semantics/common/compat.k

```diff
@@ -81,8 +81,81 @@ module COMPAT-SYNTAX
 
 endmodule
 
+module COMPAT-KAST [kast]
```

Comment:
Also the `[kast]` here?

Reply:
This is so this module is not imported on the llvm backend.